simd/unit_tests/include/SIMDTesting_Ops.hpp

```diff
@@ -129,4 +129,180 @@ class shift_left {
   }
 };
 
+class cbrt_op {
+ public:
+  template <typename T>
+  auto on_host(T const& a) const {
+    // FIXME specialization only exists in namespace Kokkos::Experimental for
+    // AVX2 and AVX512, the fallback is in namespace Kokkos
+#if (defined(KOKKOS_COMPILER_INTEL) || defined(KOKKOS_COMPILER_INTEL_LLVM)) && \
+    (defined(KOKKOS_ARCH_AVX2) || defined(KOKKOS_ARCH_AVX512XEON))
+    using abi_type = typename T::abi_type;
+    if constexpr (!std::is_same_v<abi_type,
+                                  Kokkos::Experimental::simd_abi::scalar>)
+      return Kokkos::Experimental::cbrt(a);
+#endif
+    return Kokkos::cbrt(a);
+  }
+  template <typename T>
+  auto on_host_serial(T const& a) const {
+    return Kokkos::cbrt(a);
+  }
+};
+
+class exp_op {
+ public:
+  template <typename T>
+  auto on_host(T const& a) const {
+    // FIXME specialization only exists in namespace Kokkos::Experimental for
+    // AVX2 and AVX512, the fallback is in namespace Kokkos
+#if (defined(KOKKOS_COMPILER_INTEL) || defined(KOKKOS_COMPILER_INTEL_LLVM)) && \
+    (defined(KOKKOS_ARCH_AVX2) || defined(KOKKOS_ARCH_AVX512XEON))
+    using abi_type = typename T::abi_type;
+    if constexpr (!std::is_same_v<abi_type,
+                                  Kokkos::Experimental::simd_abi::scalar>)
+      return Kokkos::Experimental::exp(a);
+#endif
+    return Kokkos::exp(a);
```

Comment:
You can really only use math functions in the `Experimental` namespace with `Kokkos_ENABLE_DEPRECATED_CODE_4=OFF`. Something like
```suggestion
#ifdef KOKKOS_ENABLE_DEPRECATED_CODE_4
      return Kokkos::Experimental::exp(a);
#else
      return Kokkos::exp(a);
#endif
```
or even
```suggestion
      return Kokkos::exp(a);
```
should fix the CI.

Reply:
Applied the first version of suggestion to `cbrt`, `exp` and `log`.